docs/en/getting-started/create-or-restore-your-status-profile-using-a-recovery-phrase.md

```diff
@@ -1,16 +1,97 @@
 ---
 id: 374
 revision: 0
+revision: o
 language: en
 hide:
   - navigation
 ---
 
 # Create or restore your Status profile using a recovery phrase
 
- :octicons-tools-24: In progress
+You can link your existing Ethereum address to a new Status profile. For instance, if you already have a [non-custodial Ethereum wallet][how-to-use-wallet-your-quick-start-guide] such as Metamask or Trust Wallet, you already have an Ethereum address and a [recovery phrase][understand-your-status-keys-and-recovery-phrase].
 
-!!! note ""
-     We're working on this content.
+## What to expect
 
---8<-- "includes/urls-en.txt"
\ No newline at end of file
+- Your wallet funds are safe and recoverable as long as you know your recovery phrase and [keep it safe][back-up-and-secure-your-recovery-phrase].
+- You can create a new Status profile using your existing Ethereum address.
+- If you already use Status in one device and want to add your profile and funds to a new device, you don't need your recovery phrase. [Sync your new device][add-a-new-device-to-your-status-profile] for the best experience.
+- If you already use Status but don't have access to your previous device, you can restore your Status profile on a new device if you have used the app within the past 30 days.
+- If you haven't used Status in the past 30 days, your funds are safe, but your profile is unrecoverable.
```

Comment:
Does "unrecoverable" mean "can't restore"? And I'll be more specific, adding "Create a new profile (or link the address to your other Status profile?) if you want to continue operating the funds using the Status Wallet."

Reply:
Precisely. If you don't have your device and haven't used Status in the past 30 days, you can't restore your profile.

You can still use your recovery phrase to create a new profile, but your contact, profile configuration, etc. won't be restored from your previous profile. I will clarify this scenario in the doc.